content/en/docs/rules/overriding.md

```diff
@@ -0,0 +1,323 @@
+---
+title: Overriding Rules
+description: Overriding Falco rules
+linktitle: Overriding Rules
+aliases: 
+- /docs/rules/appending
+weight: 50
+---
+## Overview
+There may be cases where you need to adjust the behavior of the Falco-supplied {{< glossary_tooltip text="list" term_id="lists" >}}, {{< glossary_tooltip text="macro" term_id="macros" >}}, and  {{< glossary_tooltip text="rule" term_id="rules" >}}.
+
+You can override (modify) rules in falco two different ways: 
+
+1) Define multiple rules files. The additional rules files can be used to add new lists, macros and rules or to override existing ones.
```

Comment:
It doesn't fit in here but maybe you have an idea @mikegcoleman how to give a more concrete example. Somehow stating that a typical use case is to fetch the upstream Falco rules hosted in the rules repo and then customize them using the override feature.